### ISSUE TITLE
GAS V8 runtime bug

### ISSUE THREAD

**Issue by Rohed** [RomainVialard/FirebaseApp#25]
Switched to V8 Runtime and getting a bug where when, requesting to an endpoint which would normally return 'null' the whole script crashes with the 'Error connecting to Google Server'.

Well I loaded the library locally and this fix seems to do the job
baseClass_.getData = function (path, optQueryParameters) {
  // Send request
  // noinspection JSAnnotator
  var [res] = FirebaseApp_._buildAllRequests([
    {
      method: 'get',
      path: path,
      optQueryParameters: optQueryParameters,
    },
  ], this);
  
  // Throw error
  if (res instanceof Error) {
    throw res;
  }
  
  if(res){
  return res;
  }
};

TLDR: dont return anything if the value to be returned is null.
